velox/dwio/parquet/reader/PageReader.h

```diff
@@ -68,9 +68,8 @@ class PageReader {
         chunkSize_(chunkSize),
         nullConcatenation_(pool_) {}
 
-  common::CompressionKind ThriftCodecToCompressionKind(
-      thrift::CompressionCodec::type codec) const {
-    switch (codec) {
+  common::CompressionKind thriftCodecToCompressionKind() {
```

Comment:
Why are the definitions also in the header file? Can you please move the definition to the cpp file?

Reply:
Also, does it need to be public? If not, please move the declaration to the private section. Make sure the cpp file function order is the same as the .h.